src/src/AdminBundle/Controller/EventController.php

```diff
@@ -16,13 +16,13 @@
 class EventController extends Controller
 {
     /**
-     * @Route("/event_type/edit/{id}/event/add", name="event_add")
+     * @Route("/event_type/edit/{id}/event/add", name="event_add", requirements={"id"="\d+"})
      * @Method({"GET", "POST"})
      */
     public function createAction($id, Request $request)
     {
         $repository = $this->getDoctrine()->getRepository(EventType::class);
-        $eventType = $repository->findOneBy(['id' => intval($id)]);
```

Comment:
@nitrajka, zmazal som `intval` kedze to funguje aj bez toho a nikde to nepouzivame, bol tam nejaky specialny dovod preco si ho pridala?